libs/domain/picking/src/components/user-profile/user-profile.component.ts

```diff
@@ -1,7 +1,124 @@
+import { when } from 'lit/directives/when.js';
+import { delay, switchMap, tap } from 'rxjs';
+import { OfflineDataPlugin } from '../../../offline/data-plugin';
+import { userProfileComponentStyles } from './user-profile.styles';
 
 export class UserProfileComponent extends LitElement {
+  static styles = userProfileComponentStyles;
+
+  protected routerService = resolve(RouterService);
+  protected authService = resolve(AuthService);
+  protected syncSchedulerService = resolve(SyncSchedulerService);
+
+  @state()
+  protected loading: boolean | null = null;
+
+  @asyncState()
+  protected currentRoute = valueType(this.routerService.currentRoute());
+
+  @asyncState()
+  protected route = valueType(this.routerService.route());
+
+  @asyncState()
+  protected pendingSyncs = valueType(this.syncSchedulerService.hasPending());
+
   protected override render(): TemplateResult {
-    return html` Ama composition, believe me! `;
+    const isPicking = this.route?.includes('/picking/');
+    const isMainPage = this.route === '/';
+
+    return html`
+      <div class="info-block">
+        <dl>
+          <dt class="info-label">${i18n('user.profile.employee-id')}</dt>
+          <dd class="info-value">admin@spryker.com</dd>
+        </dl>
+      </div>
+
+      ${when(
+        this.pendingSyncs && !isPicking,
+        () =>
+          html`
+            <oryx-notification type="info" scheme="dark">
+              ${i18n(
+                'user.profile.you-can’t-log-out-because-of-a-pending-synchronization'
+              )}.
+            </oryx-notification>
+          `
+      )}
+      ${when(
+        isPicking,
+        () =>
+          html`
+            <oryx-notification type="info" scheme="dark">
+              ${i18n(
+                'user.profile.you-can’t-log-out-because-picking-is-in-progress'
+              )}.
+            </oryx-notification>
+          `
+      )}
+
+      <div class="info-footer">
+        <oryx-button type="secondary" outline>
+          <button
+            ?disabled="${isPicking || this.pendingSyncs}"
+            @click=${this.onLogOut}
+          >
+            ${i18n('user.profile.log-Out')}
+          </button>
+        </oryx-button>
+
+        ${when(
+          isMainPage,
+          () =>
+            html`
+              <oryx-button ?loading=${this.loading} type="secondary" outline>
+                <button @click=${this.onReceiveData}>
+                  ${i18n('user.profile.receive-Data')}
+                </button>
+              </oryx-button>
+            `
+        )}
+      </div>
+    `;
+  }
+
+  protected onReceiveData(): void {
+    const app = resolve(AppRef);
+    const injector = app?.requirePlugin(InjectionPlugin).getInjector();
+    const injectorDataPlugin = app?.requirePlugin(OfflineDataPlugin);
+
+    injectorDataPlugin
+      .clearDb(injector)
+      .pipe(
+        tap(() => (this.loading = true)),
+        delay(500),
+        switchMap(() => injectorDataPlugin.populateDb(injector))
```

Comment:
I suggest to combine `clearDb` with `populateDb` into a single method to simplify component code.